README.md

```diff
@@ -14,6 +14,7 @@ don't naturally belong to other topic monorepos (like
 
 | Package | Description | Version |
 | --- | --- | --- |
+| [cli_config](pkgs/cli_config/) | A package to take config values from configuration files, CLI arguments, and environment variables. |
```

Comment:
nit: you can end the markdown table here w/ ` | |` (an empty column for the pub badge)